calico/getting-started/kubernetes/self-managed-onprem/onpremises.mdx

```diff
@@ -95,6 +95,14 @@ The option, **Kubernetes API datastore, more than 50 nodes** provides scaling us
 
 #### Install Calico with Kubernetes API datastore, 50 nodes or less
 
+:::note
+
+This option is maintained for upgrade compatibility but we recommend that new clusters use the operator, which
+will automatically configure {{prodname}} correctly for your cluster size (including deploying the Typha scale-out
+proxy and securing it when necessary).
+
```

Comment:
Hmm, should we go further and just delete these sections?  It sounds like there are no remaining cases where anyone should use them.

Reply:
I think there are a lot of folks who just upgrade by splatting the next version of these manifests onto their cluster.  Actually removing them requires us to be ready for the "you broke my workflow" backlash and to commit some resource to plugging gaps in the operator as folks try to migrate.

I felt comfortable ratcheting up the "anti recommendation" but thought I'd need more buy in to remove the sections entirely.

Reply:
OK, thanks.

Reply:
That probably won't stop people using manifest install.  AFAICT most users don't ever find our docs, they just follow whatever k8s tutorial blog takes their fancy.

And that's the danger of removing support for manifest install - suddenly all the blogs online that talk about installing Calico don't work, so they give up and find a different blog that recommends a different CNI

Reply:
Maybe also add a link to the operator migration doc?

Reply:
To be clear, I was only suggesting removal of this doc, not of code-level support for manifest install.

Also, though, if I was searching for "Calico install", I'd much prefer to find and follow the official documentation than a random 3rd party blog, and I guess that would be the same for most people.  So if "Calico install" doesn't find our docs, or if there's something about our docs that makes someone go looking for an alternative source, we should investigate that.